src/tools/builder/scripts/scripts.js

```diff
@@ -11,7 +11,12 @@ module.exports = (input, dest, options) => {
     options.uglify === true ||
     (options.uglify !== false && process.env.NODE_ENV === 'production');
 
-  const uglifyOptions = {};
+  // ECL uses Pikaday for the datepicker component. Pikaday dynamically requires moment.js which messes up JS bundling.
+  // ECL does not want to include moment.js in its release in order to reduce the final bundle size.
+  // Instruct minifier to preserve the UMD locally scoped 'moment' (default Pikaday module) variable in Pikaday in order to correctly reference the global 'moment' included separately from the ECL library bundle.
+  // When Pikaday really removes moment from its dependencies and does not load it dynamically, bundlers such as rollup will be able to handle this more gracefully.
+  // @see https://github.com/Pikaday/Pikaday/issues/815
```

Comment:
So this means that for using the component the consumers need to add moment.js on their side, right? Shouldn't we add this information in the api.mdx file, then?

Reply:
Documentation added